packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/TableSelector.tsx

```diff
@@ -44,6 +44,16 @@ export function TableSelector({ tables }: { tables?: Table[] }) {
     }
   }, [selectedTableId, setTableId, tables]);
 
+  useLayoutEffect(() => {
+    if (open && selectedTableId && selectedTableConfig) {
+      setTimeout(() => {
```

Comment:
Don't like it but couldn't get it to work any other way. Need to wait for dropdown to get rendered in order for it to be queried.